doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
+
+This feature is in a very early phase, at this moment you can create, delete, modify and update entities and you can subscribe for notifications, but is not well tested and up to date is only a wrapped version of REST API, there is not a binary protocol.
+
+## How I can use it?
+
+In the websocket protocol we need wrap some parts of REST API in JSON format in order to get a complete websocket message, to do this
+we can use the following JSON template:
+
+```
+{
+    "verb": "HTTP verb",
+    "url": "/uri/used/to/this/operation",
+    "params": {"key": "value"},
+    "headers": {"key": "value"},
+    "payload" : {Put a normal REST message here}
+}
+```
+Description:
+
+* verb: is the HTTP verb used to this operation.
+
+* url: is the URL used in this operation, for example: `/v2/entities` or `/v2/subscriptions`
+
+* params: these are the options pased normally in the URL after the `?` symbol and they are a key-value pair. For example if you use `http://server/v2?options=keyValues` on websockets you must use `"params": {"options": "keyValues"}`
+
+* headers: these are the headers sent to server in a REST message, you can use this to sent information about the tenant, servicePath, etc.
+
+* payload: the payload is a valid JSON with the REST message to be processed.
```

Comment:
"the payload is a valid JSON with the REST message to be processed, according to the specific operation as described in the REST API specification."